### ISSUE TITLE
Failed to  build iOS

### ISSUE THREAD

**Issue by pivalig** [dpa99c/cordova-plugin-firebasex#360]
# Bug report

**Current behavior:**
I trying to build a Test project for cordova-plugin-firebasex  and get this error:
`InAppMessagingDisplayResources.bundle" not found. Run 'pod install'`

**Expected behavior:**
To have a successfully build

**Steps to reproduce:**

Just follow the steps described in https://github.com/dpa99c/cordova-plugin-firebasex-test
**Screenshots**

<img width="1344" alt="Screenshot 2020-04-07 at 08 56 41" src="https://user-images.githubusercontent.com/6332617/78635170-44139c80-78ae-11ea-9a90-20b09a194f51.png">
**Environment information**

- Cordova CLI version 
	- `cordova -v`
9.0.0 (cordova-lib@9.0.1)
- Cordova platform version
	- `cordova platform ls`
Installed platforms:
  ios 5.1.1
Available platforms: 
  android ^8.0.0
  browser ^6.0.0
  electron ^1.0.0
  osx ^5.0.0
  windows ^7.0.0
- Plugins & versions installed in project (including this plugin)
    - `cordova plugin ls`
cordova-custom-config 5.1.0 "cordova-custom-config"
cordova-plugin-androidx 1.0.2 "cordova-plugin-androidx"
cordova-plugin-androidx-adapter 1.1.0 "cordova-plugin-androidx-adapter"
cordova-plugin-customfcmreceiver 1.0.0 "Custom FCM Receiver"
cordova-plugin-dialogs 1.3.2-dev "Notification"
cordova-plugin-enable-multidex 0.2.0 "Enable Multidex"
cordova-plugin-firebasex 9.0.1 "Google Firebase Plugin"
cordova-plugin-whitelist 1.3.4 "Whitelist"
- Dev machine OS and version, e.g.
    - OSX
        - `sw_vers`
ProductName:    Mac OS X
ProductVersion: 10.15.4
BuildVersion:   19E266
    - Windows 10
        - `winver`
        
_Runtime issue_
- Device details
    - _e.g. iPhone X, Samsung Galaxy S8, iPhone X Simulator, Pixel XL Emulator_
- OS details
    - _e.g. iOS 12.2, Android 9.0_	
	
_Android build issue:_	
- Node JS version
    - `node -v`
- Gradle version
	- `ls platforms/android/.gradle`
- Target Android SDK version
	- `android:targetSdkVersion` in `AndroidManifest.xml`
- Android SDK details
	- `sdkmanager --list | sed -e '/Available Packages/q'`
	
_iOS build issue:_
- Node JS version
    - `node -v`
v12.16.1
- Xcode version
Version 11.4 (11E146)

**Related code:**
```
insert any relevant code here such as plugin API calls / input parameters
```

**Console output**
<details>
<summary>console output</summary>

```

// Paste any relevant JS/native console output here

```

</details><br/><br/>

**Other information:**

**Comment by dpa99c** [comment]
Duplicate of #326.
See [the documentation](https://github.com/dpa99c/cordova-plugin-firebasex#cordova-cli-builds) regarding this.
TL;DR: use `cordova-plugin-firebasex@9.0.1-cli` when building from the CLI
